core/frontend/src/DisplayStyleState.ts

```diff
@@ -409,20 +409,33 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
   }
 
   /** @internal */
-  public changeBaseMapProps(props: MapLayerProps | ColorDef) {
+  public changeBaseMapProps(props: Partial<MapLayerProps>| ColorDef): boolean {
+    let changed = true;
     if (props instanceof ColorDef) {
       const transparency = this.settings.mapImagery.backgroundBase instanceof ColorDef ? this.settings.mapImagery.backgroundBase.getTransparency() : 0;
       this.settings.mapImagery.backgroundBase = props.withTransparency(transparency);
     } else {
       if (this.settings.mapImagery.backgroundBase instanceof MapLayerSettings)
         this.settings.mapImagery.backgroundBase = this.settings.mapImagery.backgroundBase?.clone(props);
       else {
-        const backgroundLayerSettings = BaseMapLayerSettings.fromJSON(props);
-        if (backgroundLayerSettings)
-          this.settings.mapImagery.backgroundBase = backgroundLayerSettings;
+        changed = false;
       }
     }
 
+    if (changed) {
+      this._synchBackgroundMapImagery();
+    }
+    return changed;
+  }
+
+  public setBaseMapProps(props: MapLayerProps| ColorDef) {
```

Comment:
@pmconne 'Workaround' to solve the `changeBaseMapProps `problem mentioned below.  This method should be mainly used to switch between a color baselayer and a MapLayerSettings baselayer.  I feel like we could do better, but this dual type baseLayer is limiting our options.